backend/src/baserow/contrib/integrations/local_baserow/api/serializers.py

```diff
@@ -54,9 +53,8 @@ def to_internal_value(self, data):
 
 
 class LocalBaserowTableServiceFilterSerializer(serializers.ModelSerializer):
-    value = OptionalFormulaSerializerField(
+    value = FormulaSerializerField(
```

Comment:
We use a regular formula field and we always have formula now. Thi compatibility with old values is handled in the frontend and during the resolution now. It also uses the `raw` formula type.

Reply:
Yeah that's expected but it shouldn't break at least.